src/gui/MainWindow.cpp

```diff
@@ -2320,6 +2322,14 @@ void MainWindow::RecreateMenu()
 		SetMenuVisible(false);
 }
 
+void MainWindow::UpdateChildWindowTitleRunningState()
+{
+	const bool running = CafeSystem::IsTitleRunning();
+
+	if(auto graphicPacksWindow = dynamic_cast<GraphicPacksWindow2*>(m_graphic_pack_window))
```

Comment:
Rather than dynamic casting, we should change `m_graphic_pack_window` to be of type `GraphicPacksWindow2*`